float64 x
float64 y
float64 theta
float64 v
float64 w
---
float64[] x_poses
float64[] y_poses
